Minio.Examples/Cases/ListObjects.cs

```diff
@@ -25,18 +25,21 @@ internal static class ListObjects
         string bucketName = "my-bucket-name",
         string prefix = null,
         bool recursive = true,
-        bool versions = false)
+        bool versions = false,
+        bool includeUserMetadata = false)
     {
         try
         {
             Console.WriteLine("Running example for API: ListObjectsAsync");
             var listArgs = new ListObjectsArgs()
                 .WithBucket(bucketName)
                 .WithPrefix(prefix)
-                .WithRecursive(recursive);
+                .WithRecursive(recursive)
+                .WithVersions(versions)
+                .WithUserMetadata(includeUserMetadata);
             var observable = minio.ListObjectsAsync(listArgs);
             var subscription = observable.Subscribe(
-                item => Console.WriteLine($"Object: {item.Key}"),
+                item => Console.WriteLine($"Object: {item.Key}, content-type: {item.UserMetadata.ToList()[0].Value}"),
```

Comment:
This will result in an exception if there is no meta-data. The `UserMetadata` is already a list, so there is no need to call `ToList()`.

Reply:
N/A
This line is removed.

Reply:
@ramondeklein,

Done. Please check it out.